drivers/i3c/i3c_shell.c

```diff
@@ -2104,29 +2105,28 @@ SHELL_DYNAMIC_CMD_CREATE(dsub_i3c_device_name, i3c_device_name_get);
 
 #ifdef CONFIG_I3C_USE_IBI
 /* L2 I3C IBI Shell Commands*/
-SHELL_STATIC_SUBCMD_SET_CREATE(
-	sub_i3c_ibi_cmds,
-	SHELL_CMD_ARG(hj, &dsub_i3c_device_name,
-		      "Send IBI HJ\n"
-		      "Usage: ibi hj <device>",
-		      cmd_i3c_ibi_hj, 2, 0),
-	SHELL_CMD_ARG(tir, &dsub_i3c_device_name,
-		      "Send IBI TIR\n"
-		      "Usage: ibi tir <device> [<byte1>, ...]",
-		      cmd_i3c_ibi_tir, 2, MAX_I3C_BYTES),
-	SHELL_CMD_ARG(cr, &dsub_i3c_device_name,
-		      "Send IBI CR\n"
-		      "Usage: ibi cr <device>",
-		      cmd_i3c_ibi_cr, 2, 0),
-	SHELL_CMD_ARG(enable, &dsub_i3c_device_attached_name,
-		      "Enable receiving IBI from target\n"
-		      "Usage: ibi enable <device> <target>",
-		      cmd_i3c_ibi_enable, 3, 0),
-	SHELL_CMD_ARG(disable, &dsub_i3c_device_attached_name,
-		      "Disable receiving IBI from target\n"
-		      "Usage: ibi disable <device> <target>",
-		      cmd_i3c_ibi_disable, 3, 0),
-	SHELL_SUBCMD_SET_END /* Array terminated. */
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_i3c_ibi_cmds,
+			       SHELL_CMD_ARG(hj, &dsub_i3c_device_name,
+					     "Send IBI HJ\n"
+					     "Usage: ibi hj <device>",
+					     cmd_i3c_ibi_hj, 2, 0),
+			       SHELL_CMD_ARG(tir, &dsub_i3c_device_name,
+					     "Send IBI TIR\n"
+					     "Usage: ibi tir <device> [<byte1>, ...]",
+					     cmd_i3c_ibi_tir, 2, MAX_I3C_BYTES),
+			       SHELL_CMD_ARG(cr, &dsub_i3c_device_name,
+					     "Send IBI CR\n"
+					     "Usage: ibi cr <device>",
+					     cmd_i3c_ibi_cr, 2, 0),
+			       SHELL_CMD_ARG(enable, &dsub_i3c_device_attached_name,
+					     "Enable receiving IBI from target\n"
+					     "Usage: ibi enable <device> <target>",
+					     cmd_i3c_ibi_enable, 3, 0),
+			       SHELL_CMD_ARG(disable, &dsub_i3c_device_attached_name,
+					     "Disable receiving IBI from target\n"
+					     "Usage: ibi disable <device> <target>",
+					     cmd_i3c_ibi_disable, 3, 0),
+			       SHELL_SUBCMD_SET_END /* Array terminated. */
 );
```

Comment:
Don't mix formatting changes with other changes. 
Please do it in a dedicated commit if required.